pkgs/servers/sql/postgresql/pgjwt/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchgit}:
```

Comment:
Nit: please add some whitespace after the first and before the last brace in the
parameter list and around `=` in the `name` and `version` attributes below.